app/build.gradle

```diff
@@ -97,7 +101,14 @@ dependencies {
     compile "com.android.support:recyclerview-v7:${rootProject.ext.androidSupportSdkVersion}"
     compile "com.squareup.leakcanary:leakcanary-android:1.3.1"
 
-    apt 'com.google.dagger:dagger-compiler:2.0.1'
+    apt 'com.google.dagger:dagger-compiler:2.5'
+    apt "com.ryanharter.auto.value:auto-value-gson:0.3.1"
+    apt "com.google.auto.value:auto-value:1.2"
+    // used for AndroidStudio sync
+    provided "com.google.auto.value:auto-value:1.2"
+    // to make OkBuck work with SqlDelight
+    provided 'com.github.okbuilds:SqlDelightBin:0.4.3'
```

Comment:
do we have to maintain this everytime a new version of sqldelight is released?

This does not seem ideal. We should open a PR on sqldelight so they can upload another artifact that does what we want so their changes will keep the binary compatible